package.json

```diff
@@ -59,7 +59,6 @@
     "@semantic-release/git": "^10.0.1",
     "@types/jsdom": "^21.1.7",
     "@types/node": "^20.15.0",
-    "@types/node-emoji": "^2.1.0",
```

Comment:
Removing deprecated dependency (types are now included in the `node-emoji` package.